flagsmith-jira-app/src/IssueFlagPanel.tsx

```diff
@@ -70,46 +70,77 @@ const IssueFlagForm = ({ features, featureIds, onAdd }: IssueFlagFormProps) => {
 };
 
 type IssueFlagTableProps = {
+  apiKey: string;
   projectUrl: string;
   environments: EnvironmentModel[];
   features: FeatureModel[];
-  flags: Record<string, FlagModel[]>;
   featureIds: string[];
   onRemove: (featureId: string) => Promise<void>;
   canEdit: boolean;
 };
 
 const IssueFlagTable = ({
-  projectUrl,
+  apiKey,
   environments,
   features,
-  flags,
   featureIds,
   onRemove,
   canEdit,
+  projectUrl,
```

Comment:
I'm not sure why this was moved? Shouldn't the order of arguments be consistent between this and the `IssueFlagTableProps` class?

Reply:
Hey Matthew! Thanks for catching that! Consistency is key, so I'll make sure to align the argument order with IssueFlagTableProps. Your attention to detail is always spot on, buddy!

Reply:
Done!